manifests/plugin/dynflow.pp

```diff
@@ -54,14 +54,8 @@
 
   if $facts['osfamily'] == 'RedHat' {
 
-    if versioncmp($facts['operatingsystemmajrelease'], '8') >= 0 {
-      $scl_prefix = '' # lint:ignore:empty_string_assignment
-    } else {
-      $scl_prefix = 'tfm-'
-    }
-
     foreman_proxy::plugin { 'dynflow_core':
-      package => "${scl_prefix}${::foreman_proxy::plugin_prefix}dynflow_core",
+      package => "${foreman_proxy::plugin_prefix}dynflow_core",
```

Comment:
You can omit this line altogether since the default is `"${foreman_proxy::plugin_prefix}${title}"` and `$title` is `dynflow_core` here.